### ISSUE TITLE
PNI product page: updates to MMS section

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#4041]
Based on user testing, we have come up with an easier to understand Minimum Security Standards section.

### List of updates
- [ ] Add background background `Gray 05 ($gray-05) #f2f2f2`
- [ ] Add mini-badge next to title
- [ ] Add title `Overall Security Rating`
- [ ] Hide all the stars
- [ ] Add a number value to star rating (example: ★★★☆☆ will now be ⅗ ★) 
- [ ] Update `Yes` = `✓`, `No` = `X`, and keep `n/a` = `N/A` [Minimum-Security-Assets.zip](https://github.com/mozilla/foundation.mozilla.org/files/3963111/Minimum-Security-Assets.zip)

### Comp for desktop
![MSS Section@2x](https://user-images.githubusercontent.com/20977376/71192306-071b8f80-2256-11ea-940b-419615f75854.jpg)

### Comp for mobile
<img width="215" alt="image" src="https://user-images.githubusercontent.com/1068783/70482775-4a7a3f00-1a9b-11ea-81a5-4e07ed50ea55.png">

**🗒  Design -> Dev Handoff Checklist**

- [x] Design has been finalized
- [x] Link to original design ticket - _#4016_
- [x] Link to Redpen - _https://redpen.io/p/mn3629c03f84209e3b_ update once #4040 is done
- [x] Design changes been merged into the master Sketch file _#4040_
- [x] Desktop and mobile mockups are included
- [x] Assets are included: [Minimum-Security-Assets.zip](https://github.com/mozilla/foundation.mozilla.org/files/3963111/Minimum-Security-Assets.zip)
- [x] Hover state for elements are designed - _N/A_
- [x] Other states for elements are designed (e.g., error state for form) - _N/A_

cc @beccaklam

**Comment by beccaklam** [comment]
@mmmavis All the designs are in! This is ready to start on :)

**Comment by mmmavis** [comment]
@beccaklam what symbol should represent "Unknown"?

e.g., https://foundation.mozilla.org/en/privacynotincluded/products/ring-indoor-cam/

**Comment by beccaklam** [comment]
Ah, the question mark! I updated the sketch file but not the redpen, will do that now. The question mark asset is included in the zip file above.

> @beccaklam what symbol should represent "Unknown"? Should it stay as "Unknown"?
> 
> e.g., https://foundation.mozilla.org/en/privacynotincluded/products/ring-indoor-cam/
